### PR TITLE
Added comments to config_example.py

### PR DESCRIPTION
**Pull request by konnovdev**
I figured it would be nice to have explanations in config_example.py, since it's supposed to be a user guide

### CODE REVIEW

File: config_example.py [hunk 1]
```diff
@@ -18,18 +24,32 @@
...
+                                 "<b>Usage</b>:\n"
+                                 "- send a message that only contains a link of a YouTube video "
+                                 "or a SoundCloud track.\n"
+                                 "- Playlists are not supported, but you can select multiple "
+                                 "audio messages and forward them to create playlist instead.\n"
+                                 "- Your message will be deleted in private chat after the music "
+                                 "gets successfully uploaded.\n"
+                                 "- You can get YouTube links with inline bot @vid and use "
+                                 "them.\n\n" +
              CONTACTS_HTML)
```
Comment (dashezup) [inline config_example.py:40]: Here it exceed 79 characters length.

**Review by dashezup — CHANGES_REQUESTED** [review]

File: config_example.py (hunk 1 above)
Comment (dashezup) [reply config_example.py:40]: I think both style is fine (as long as it does not exceed 79 characters), but the string was too long, it waste a lot of space to just indent another level.
If you want it to look better, maybe just this?
```
HELP_HTML = (AVAILABILITY_HTML +
             "\n\n"
             "<b>Usage</b>:\n"
             "- send a message that only contains a link of a YouTube video "
             "or a SoundCloud track.\n"
             "- Playlists are not supported, but you can select multiple "
             "audio messages and forward them to create playlist instead.\n"
             "- Your message will be deleted in private chat after the music "
             "gets successfully uploaded.\n"
             "- You can get YouTube links with inline bot @vid and use "
             "them.\n\n" +
             CONTACTS_HTML)
```

File: config_example.py [hunk 2]
```diff
@@ -18,18 +24,32 @@
               "For now it downloads music from YouTube or SoundCloud "
               "whenever you send the link and makes new members go through "
               "captcha." + "\n\n"
-              "<i>Source code available at cpop.tw/code under GPLv3 "
-              "license</i>\n\n" +
+                           "<i>Source code available at cpop.tw/code under GPLv3 "
+                           "license</i>\n\n" +
               CONTACTS_HTML)
```
Comment (dashezup) [inline config_example.py:29]: This indent also looks not necessarily, should we just remove unnecessary `+`?

**Comment by dashezup** [comment]
Do you think that it would be too annoying that CONTACTS_HTML includes your `@username` which mentions you? (you will receive notification) we can change that to `<a href="https://t.me/konnov">ilya</a>`

**Comment by dashezup** [comment]
Pushed new commit to apply my suggestions.
added `disable_web_page_preview=True` for `/about` and `/help`.
fwiw, the difference between t.me link and `@mention` is that the former one open a PM directly and the latter one open user profile instead.

If you don't like some of my changes, just change it back and commit. when you think it's ok then just "Squash and merge"

File: config_example.py [hunk 3]
```diff
@@ -1,15 +1,21 @@
...
+DEV_ID = 2011545269  # errors will be forwarded to this user ID
+
+# Members of these chats can use the bot in private.
+# Group chat IDs always start with `-`
 PROMOTION_LIST_CHAT_ID = (
     -1001853277651,
     -1001748900794
-    )
+)
+
```
[the remark this answers is not in the document]
Comment (konnovdev) [reply config_example.py:12]: actually this is auto formatting in pycharm (Option + Command + L)

but it's default code style, i didn't change anything

Screenshot 2020-12-21 at 00 54 58

so i guess if they made it this way by default, it's more common

[the remark this answers is not in the document]
Comment (konnovdev) [reply config_example.py:12]: looks like Python library **logging** also uses the same style, although here it's different brackets

Screenshot 2020-12-21 at 00 59 22

[the remark this answers is not in the document]
Comment (konnovdev) [reply config_example.py:12]: so i would go for
```
PROMOTION_LIST_CHAT_ID = (
    -1001853277651,
    -1001748900794
)
```
over
```
PROMOTION_LIST_CHAT_ID = (
    -1001853277651,
    -1001748900794
    )
```

i think first one looks prettier too

File: config_example.py (hunk 1 above)
Comment (konnovdev) [reply config_example.py:40]: oh shit i forgot to check lint

Comment (konnovdev) [reply config_example.py:40]: actually looks like lint didn't say anything
Screenshot 2020-12-21 at 01 05 10

Comment (konnovdev) [reply config_example.py:40]: but i'll fix it tomorrow, i agree it looks too long

**Comment by konnovdev** [comment]
`If you don't like some of my changes`
looks good

this is the only question https://github.com/konnovdev/cpop_bot/pull/16#discussion_r546409456

File: config_example.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (konnovdev) [reply config_example.py:12]: `I think at least we should avoid having both styles in one project.`

i think it should also be possible to add ident check to the lint

File: .github/workflows/python-package.yml [hunk 4]
```diff
@@ -28,5 +28,5 @@ jobs:
       run: |
         # stop the build if there are Python syntax errors or undefined names
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-        # exit-zero treats all errors as warnings
-        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=79 --statistics
+        # no --exit-zero
```
Comment (konnovdev) [inline .github/workflows/python-package.yml:31]: `# no --exit-zero` - i don't think we need this comment, I'll remove it

**Comment by konnovdev** [comment]
Just tested by adding a super long line:

Screenshot 2020-12-21 at 11 35 30

Build fails as expected: 

Screenshot 2020-12-21 at 11 35 12

File: .github/workflows/python-package.yml (hunk 4 above)
Comment (dashezup) [reply .github/workflows/python-package.yml]: Ok, after change this, this PR may be merged.

**Pull request merged**